### ISSUE TITLE
[Bug]: Null PlayerTeam on entity causes crash

### ISSUE THREAD

**Issue by pat277** [FTBTeam/FTB-Mods-Issues#291]
### Mod

FTB Chunks

### Mod version

FTB-Chunks-Fabric 1802.3.6-build.145

### Forge / Fabric version

1.18.2-1.14.4

### What issue are you having?

Crashes with the (specific) use of Mo'Enchantments mod, specifically the LivingItemEntity which adds some amount of automation

### Crashlogs

https://paste.feed-the-beast.com/view/48d2697f

### Steps to reproduce

Mo' Enchantments adds an entity which is accessible with a tool-only enchantment. The problem is it adds some automation, which does not obey FTB-Chunks ownership. I am sorry, but that is all the information I have.

**Comment by Raidobw2** [comment]
Thanks for the report!

I'll add the relevant stacktrace directly here as well
```
java.lang.NullPointerException: Cannot invoke "dev.ftb.mods.ftbteams.data.PlayerTeam.getExtraData()" because the return value of "dev.ftb.mods.ftbteams.data.TeamManager.getInternalPlayerTeam(java.util.UUID)" is null
        at dev.ftb.mods.ftbchunks.data.ClaimedChunkManager.getBypassProtection(ClaimedChunkManager.java:143)
        at dev.ftb.mods.ftbchunks.data.ClaimedChunkManager.protect(ClaimedChunkManager.java:172)
        at dev.ftb.mods.ftbchunks.FTBChunks.blockBreak(FTBChunks.java:272)
        at net.minecraft.server.network.ServerPlayerInteractionManager.handler$zci000$onBreak(ServerPlayerInteractionManager:550)
        at net.minecraft.server.network.ServerPlayerInteractionManager.tryBreakBlock(ServerPlayerInteractionManager:244)
        at com.biom4st3r.moenchantments.entities.LivingItemEntity.breakBlockAsPlayer(LivingItemEntity.java:155)
        at com.biom4st3r.moenchantments.entities.FindOreGoal2.tick(FindOreGoal2.java:226)
        at net.minecraft.entity.ai.goal.PrioritizedGoal.tick(PrioritizedGoal:65)
        at net.minecraft.entity.ai.goal.GoalSelector.tickGoals(GoalSelector:119)
        at net.minecraft.entity.ai.goal.GoalSelector.tick(GoalSelector:110)
        at net.minecraft.entity.mob.MobEntity.tickNewAi(MobEntity:742)
        at net.minecraft.entity.LivingEntity.tickMovement(LivingEntity:2609)
        at net.minecraft.entity.mob.MobEntity.tickMovement(MobEntity:546)
        at net.minecraft.entity.passive.PassiveEntity.tickMovement(PassiveEntity:127)
        at net.minecraft.entity.passive.AnimalEntity.tickMovement(AnimalEntity:58)
        at net.minecraft.entity.LivingEntity.tick(LivingEntity:2354)
        at net.minecraft.entity.mob.MobEntity.tick(MobEntity:346)
        at com.biom4st3r.moenchantments.entities.LivingItemEntity.tick(LivingItemEntity.java:388)
        at net.minecraft.server.world.ServerWorld.tickEntity(ServerWorld:702)
        at net.minecraft.world.World.tickEntity(World:485)
        at net.minecraft.server.world.ServerWorld.method_31420(ServerWorld:363)
        at net.minecraft.world.EntityList.forEach(EntityList:54)
        at net.minecraft.server.world.ServerWorld.tick(ServerWorld:331)
        at net.minecraft.server.MinecraftServer.tickWorlds(MinecraftServer:907)
        at net.minecraft.server.dedicated.MinecraftDedicatedServer.tickWorlds(MinecraftDedicatedServer:322)
        at net.minecraft.server.MinecraftServer.tick(MinecraftServer:851)
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer:697)
        at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer:273)
        at java.lang.Thread.run(Thread.java:833)
```

**Comment by Raidobw2** [comment]
Linked with https://github.com/FTBTeam/FTB-Mods-Issues/issues/196, though 196 is about a Formation Plane from AE2

**Comment by desht** [comment]
Fixed in 1.16/1.18/1.19
